spec/requests/expenses_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
```

Comment:
- [ ] Excellent adding unit tests for all the most important components of the RoR application and integration tests with capybara. 👍🏻 However, I get error messages about ` Bullet.enable` Please correct this 👍🏻

![image](https://user-images.githubusercontent.com/79058364/224012615-acaa1fc5-d727-4b5d-9d20-86dc58f48316.png)